python_modules/dagster/dagster/_daemon/daemon.py

```diff
@@ -230,18 +230,28 @@ def core_loop(
 
 
 class IntervalDaemon(DagsterDaemon[TContext], ABC):
-    def __init__(self, interval_seconds, jitter_seconds: float = 0):
+    def __init__(
+        self,
+        interval_seconds,
+        *,
+        interval_jitter_seconds: float = 0,
+        initial_jitter_seconds: float = 0,
+    ):
         self.interval_seconds = check.numeric_param(interval_seconds, "interval_seconds")
-        self.jitter_seconds = jitter_seconds
+        self.interval_jitter_seconds = interval_jitter_seconds
+        self.initial_jitter_seconds = initial_jitter_seconds
```

Comment:
`startup_jitter_seconds`? Just because `interval` and `initial` read so similarly when skimming.